store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/AccumuloPropertiesTest.java

```diff
@@ -54,6 +56,86 @@ public void shouldMergeAccumuloJsonModulesAndDeduplicate() {
         assertEquals(SketchesJsonModules.class.getName() + "," + TestCustomJsonModules1.class.getName(), modules);
     }
 
+
+    @Test
+    public void getEnabledKerberosShouldReturnFalseIfSetFalse() {
+        // Given
+        final AccumuloProperties props = new AccumuloProperties();
+        final boolean ENABLE_KERBEROS = false;
```

Comment:
wrong case.

Reply:
Do you mean it being all caps? This style was copied form the preexisting test under these new ones - `shouldSetProperties`.

Reply:
I think he is suggesting only static variables should be all caps.
Should be automatically caught by check style though.

Reply:
Yeah, why isn't check style picking up on this, is something broken?

Reply:
Fixed, agree it should be picked up by checkstyle. The other instances of all caps in this class can be fixed in a different PR - probably one which adds checking for this to checkstyle.